model/trace/http.yaml

```diff
@@ -108,9 +108,14 @@ groups:
         note: >
           When [request target](https://www.rfc-editor.org/rfc/rfc9110.html#target.resource) is absolute URI, `server.port` MUST match
           URI port identifier, otherwise it MUST match `Host` header port identifier.
-      - ref: server.socket.domain
-      - ref: server.socket.address
-      - ref: server.socket.port
+      - ref: network.peer.address
+        requirement_level:
+          recommended: If different than `server.address`.
+      - ref: network.peer.port
+        requirement_level:
+          recommended: If `network.peer.address` is set.
+      - ref: proxy.address
```

Comment:
since we don't have any instrumentation that populates them and no prototypes, I wonder if we can postpone adding them to HTTP semconv for the time being.